<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Futures</class>
 <widget class="QWidget" name="Futures">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <property name="verticalSpacing">
    <number>9</number>
   </property>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>FC 代码(4</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>显示模式</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="4">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>完整轮播</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>仅价格</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>仅涨/跌幅</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>间隔。显示模式“仅价格”与“仅涨/跌幅”模式的时效性更好。</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>FC 代码(3</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>获取延迟</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Futures 代码（仅支持内盘），获取延迟表示更新数据的</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>FC 代码(2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>FC 代码(1</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>2 秒</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3 秒</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 秒</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6 秒</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8 秒</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10 秒</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="3" colspan="2">
    <widget class="QPushButton" name="pushButton_accepted">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="6" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEdit">
     <property name="placeholderText">
      <string>如：AU0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
